### PR TITLE
bq operator documentation

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

**Comment by danielnorberg** [comment]
@komamitsu PTAL

File: digdag-docs/src/operators.rst [hunk 1]
```diff
@@ -997,3 +997,580 @@ Output Parameters
...
+
+  * :command:`write_disposition: WRITE_TRUNCATE`
+  * :command:`write_disposition: WRITE_APPEND`
+  * :command:`write_disposition: WRITE_EMPTY`
+
+:command:`priority: INTERACTIVE | BATCH`
+  Specifies the priority to use for this query. *Default*: ``INTERACTIVE``.
+
+:command:`use_query_cache: BOOLEAN`
+  Whether to use BigQuery query result caching.
```
Comment (komamitsu) [inline digdag-docs/src/operators.rst]: Which is the default value?

File: digdag-docs/src/operators.rst [hunk 2]
```diff
@@ -997,3 +997,580 @@ Output Parameters
...
+  * :command:`bq>: queries/step1.sql`
+
+:command:`dataset: NAME`
+  Specifies the default dataset to use in the query and in the ``destination_table`` parameter.
+
+  * :command:`dataset: my_dataset`
+  * :command:`dataset: other_project:other_dataset`
+
+:command:`destination_table: NAME`
+  Specifies the table to store the query results in. If not specified, a new table will be automatically created to store the results.
```
Comment (komamitsu) [inline digdag-docs/src/operators.rst]: How is a new table name decided?

File: digdag-docs/src/operators.rst [hunk 3]
```diff
@@ -997,3 +997,580 @@ Output Parameters
...
+  Whether to allow arbitrarily large result tables. Requires ``destination_table`` to be set and ``use_legacy_sql`` to be true.
+
+:command:`flatten_results: BOOLEAN`
+  Whether to flatten nested and repeated fields in the query results. *Default*: ``true``. Requires ``use_legacy_sql`` to be true.
+
+:command:`use_legacy_sql: BOOLEAN`
+  Whether to use legacy BigQuery SQL. Default: ``false``.
+
+:command:`maximum_billing_tier: INTEGER`
+  Limit the billing tier for this query.
```
Comment (komamitsu) [inline digdag-docs/src/operators.rst]: What is the default value?

File: digdag-docs/src/operators.rst [hunk 4]
```diff
@@ -997,3 +997,580 @@ Output Parameters
...
+          - my_dataset_${last_session_date_compact}
+        create_tables:
+          - my_table_${session_date_compact}
+        empty_tables:
+          - my_table_${session_date_compact}
+        delete_tables:
+          - my_table_${last_session_date_compact}
+
+
+Secrets
```
Comment (komamitsu) [inline digdag-docs/src/operators.rst]: It looks like this section is already described above.

Comment (komamitsu) [inline digdag-docs/src/operators.rst]: Ah, we should write this section for each operator...?

File: digdag-docs/src/operators.rst [hunk 5]
```diff
@@ -997,3 +997,580 @@ Output Parameters
...
+  Upload the private key JSON file to the digdag server using the ``secrets`` client command:
+
+  .. code-block:: none
+
+    digdag secrets --project my_project --set gcp.credential=@my-svc-account-b4df00d.json
+
+Parameters
+~~~~~~~~~~
+
+:command:`bq_load>: TABLE | LIST`
```
Comment (komamitsu) [inline digdag-docs/src/operators.rst]: `s/TABLE/URI/`?

**Comment by komamitsu** [comment]
Great! I left some comments.

File: digdag-docs/src/operators.rst (hunk 2 above)
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: I think this documentation is incorrect...

File: digdag-docs/src/operators.rst (hunk 4 above)
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: Perhaps we should have a *Google BigQuery* section where we can have this only once...

File: digdag-docs/src/operators.rst (hunk 5 above)
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: 👍

File: digdag-docs/src/operators.rst (hunk 4 above)
Comment (danielnorberg) [inline digdag-docs/src/operators.rst]: Kept the `gcp.credential` instructions in the `bq>` operator section and linked the other bq operators to that.

**Comment by komamitsu** [comment]
👍

**Pull request merged**